### ISSUE TITLE
Alert when trying to select invalid posit

### ISSUE THREAD

**Issue by dpetrisko** [gussmith23/posit-bisection-viz#10]
Maybe a little toast when you try to put n or es into an invalid range

**Comment by mara-kr** [comment]
Or, we could have the sliders auto-correct to a good value when moving out of a valid range.

**Comment by dpetrisko** [comment]
They already do that

**Comment by dpetrisko** [comment]
The idea is to alert the user why the selection is invalid

**Comment by gussmith23** [comment]
As I described in the chat, I don't think there are actually invalid combinations!
es doesn't actually depend on n. Any non-negative value of es is meaningful and can be decoded correctly.
Obviously, n doesn't depend on anything either.
So essentially, if we just set upper and lower bounds on n and es, we should be good. I think 2<=n<=8 or 10 is ok, and then es should be 0 <= es <= 8 or 10 (it can really be anything though).
